ext/node/polyfills/tty.js

```diff
@@ -105,6 +106,25 @@ export class WriteStream extends Socket {
     this.rows = rows;
     this.isTTY = true;
   }
+
+  /**
+   * @param {number | Record<string, string>} [count]
+   * @param {Record<string, string>} [env]
+   * @returns {boolean}
+   */
+  hasColors(count, env) {
+    if (env === undefined && typeof count === "object") {
+      env = count;
+      count = 16;
+    }
+
+    // TODO(@marvinhagemeister): Ignore env parameter.
+    // Haven't seen it used anywhere, seems more done
+    // to make testing easier in Node.
+
+    const depth = op_bootstrap_color_depth();
```

Comment:
Should this be checking this for the current fd rather than using a global value?

Reply:
It looks like the answer is no to my question.